app/org/views.py

```diff
@@ -1,27 +1,62 @@
-from flask import render_template
+from flask import render_template, flash
 from flask_login import (login_required, current_user)
 from . import org
+from ..decorators import organization_required
+from .forms import OrganizationForm
+from ..models import Organization
+from .. import db
 
 
 @org.route('/')
 def index():
     return render_template('org/welcome_page.html')
 
 
-@org.route('/login')
-def org_login():
-    return render_template('org/login.html')
-
-
 @org.route('/<int:org_id>')
 def view_org(org_id):
-    return render_template('org/view_profile.html', org_id=org_id)
+    organization = Organization.query.filter_by(id=org_id).first()
+    pics = []
+    if len(organization.picture_urls) > 0:
+        pics = organization.picture_urls.split(",")
+    return render_template(
+        'org/view_profile.html', org=organization, pics=pics)
```

Comment:
if user is logged in they should be able to edit their own org's profile (aka, send current user and render an edit profile button on the frontend if they are logged in)

Reply:
We dont need to send `current_user` since that's a template global and is automatically included :) 

and I've put the edit organization profile edit link on the Organization dashboard, would that suffice?

Reply:
Done